src/components/side_panel/chart/line_bar_pie_panel/config_panel.ts

```diff
@@ -17,6 +19,10 @@ interface Props {
 interface ChartPanelState {
   datasetDispatchResult?: DispatchResult;
   labelsDispatchResult?: DispatchResult;
+  headerPosition?: {
+    direction: "row" | "column";
```

Comment:
We already have a type for that: https://github.com/odoo/o-spreadsheet/blob/master/src/types/misc.ts#L250